src/App.vue

```diff
@@ -248,6 +252,28 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
   /** The Update Current JS Date timer id. */
   private updateCurrentJsDateId = 0
 
+  /** The route breadcrumbs list. */
+  private get breadcrumbs (): Array<BreadcrumbIF> {
+    const crumbs: Array<BreadcrumbIF> = [
+      {
+        text: this.getCurrentBusinessName || 'Numbered Benefit Company',
+        href: `${sessionStorage.getItem('DASHBOARD_URL')}${this.getBusinessId}`
+      },
+      {
+        text: this.entityTitle,
+        to: { name: this.isCorrectionFiling ? RouteNames.CORRECTION : RouteNames.ALTERATION }
+      }
+    ]
+
+    if (this.isRoleStaff) {
```

Comment:
Here, too, please add a comment explaining what this code is doing / why it's needed.

Reply:
Added in latest commit

Reply:
Added!